loader/src/ui/nodes/TextInput.cpp

```diff
@@ -151,7 +151,6 @@ void TextInput::setCallback(std::function<void(std::string const&)> onInput) {
     m_onInput = onInput;
 }
 void TextInput::setEnabled(bool enabled) {
-    m_input->setMouseEnabled(enabled);

```

Comment:
To clarify, this caused the scroll issues in all lists and text input doesn't even use the mouse besides touches

Reply:
If `setTouchEnabled` is enough to make the input non-clickable then this is fine by me